debian/changelog

```diff
@@ -3,3 +3,9 @@ kano-platform-essential (4.0.0-0) unstable; urgency=low
   * Initial release of package to link all platform (RPi) dependencies
 
  -- Team Kano <dev@kano.me>  Tue, 3 Jul 2018 10:46:00 +0100
+
+kano-platform-essential (3.16.2-0) unstable; urgency=low
```

Comment:
Huh? Should this guy have the `4.0.0-0` entry from above?

Reply:
Dunno what I was thinking there... Sorry, you're completely right.